packages/matchbox/src/components/ProgressBar/styles.js

```diff
@@ -0,0 +1,40 @@
+import { tokens } from '@sparkpost/design-tokens';
+
+export const outerBase = () => `
+  position: relative;
+  background-color: ${tokens.color_gray_300};
+  border-radius: ${tokens.borderRadius_100};
+  width: 100%;
+  margin-bottom: 0.75rem;
+`;
+
+export const innerBase = () => `
+  width: 100px;
```

Comment:
I'm not sure why this is necessary? There may be a reason - if so, would you mind writing a comment describing your reasoning here?

Reply:
Not necessary, deleting